markdown/developers/tutorials/unofficial_pipelines.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Tutorial: Reusing nf-core components in non-nf-core pipelines. "
+subtitle: Guidance on how to use nf-core code and best practices in non-nf-core pipelines.
+---
+
+While we highly recommend and promote contributing to existing nf-core pipelines, in some cases this may not be possible or appropriate. In this case we hope that it will be helpful to use _parts_ of nf-core best practices for your own non-nf-core pipeline. This tutorial describes how to use nf-core code without making official nf-core pipelines.
+
+## Acknowledging nf-core
+
+If you use any form of nf-core code or infrastructure, we kindly ask that you acknowledge the nf-core initiative by adding the following markdown to your `README.md` and pipeline documentation.
+
+```markdown
+This pipeline uses code and infrastructure developed and maintained by the [nf-core](https://nf-co.re) community, reused here under the [MIT license](https://github.com/nf-core/tools/blob/master/LICENSE).
+
+> The nf-core framework for community-curated bioinformatics pipelines.
+>
+> Philip Ewels, Alexander Peltzer, Sven Fillinger, Harshil Patel, Johannes Alneberg, Andreas Wilm, Maxime Ulysse Garcia, Paolo Di Tommaso & Sven Nahnsen.
+>
+> Nat Biotechnol. 2020 Feb 13. doi: 10.1038/s41587-020-0439-x.
+>
+In addition, references of tools and data used in this pipeline are as follows:
+```
+
+## Removing nf-core only branding
+
+> Most of the following suggestions refer to when you have started your pipeline using the nf-core template.
+
+## General
+
+- Replace all references to the pipeline name that include the `nf-core/` suffix.
+    - :warning: If you still wish to use the `nf-core tools` linting functionality, this may result in a lot of linting failures. You can create a `.nf-core-lint.yml` file that allows you to ignore or skip certain lint tests. You can find more information on how to do this [here](/tools/#linting-config). One example is the `pipeline_name_conventions`
```

Comment:
```suggestion
  - :warning: If you still wish to use the `nf-core tools` linting functionality, this may result in a lot of linting failures. You can create a `.nf-core-lint.yml` file that allows you to ignore or skip certain lint tests. You can find more information on how to do this [here](/tools/#linting-config). One example is the `pipeline_name_conventions`
```